internal/api/v1/application/part.go

```diff
@@ -323,6 +325,26 @@ func fetchAppValues(c *gin.Context, logger logr.Logger, cluster *kubernetes.Clus
 	return nil
 }
 
+func fetchAppManifest(c *gin.Context, app *models.App) apierror.APIErrors {
+	m := models.ApplicationManifest{
+		ApplicationCreateRequest: models.ApplicationCreateRequest{
+			Name:          app.Meta.Name,
+			Configuration: app.Configuration,
+		},
+		Namespace: app.Meta.Namespace,
+		Origin:    app.Origin,
+		Staging:   app.Staging,
+	}
+
+	yaml, err := yaml.Marshal(m)
+	if err != nil {
+		return apierror.InternalError(err)
+	}
+
+	response.OKBytes(c, yaml)
```

Comment:
Can we use an `OKYaml` instead?

```golang
// OKYaml reports a success with some YAM data
func OKYaml(c *gin.Context, response {}interface) {
	requestctx.Logger(c.Request.Context()).Info("OK",
		"origin", c.Request.URL.String(),
		"returning", response,
	)

	c.YAML(http.StatusOK, response)
}
```

It will return the proper content type and we don't even need to marshal it.

Reply:
Will try.

Reply:
Done, commit 281c55964128a18f8a6bcf3b4d75b06e80395032